### ISSUE TITLE
Incompatibility with Python 3.7

### ISSUE THREAD

**Issue by scontini76** [pjkundert/cpppo#55]
citing https://docs.python.org/3/whatsnew/3.7.html

> [async](https://docs.python.org/3/reference/compound_stmts.html#async) and [await](https://docs.python.org/3/reference/expressions.html#await) are now reserved keywords.

that brings many issues, for example:

```
>>> import cpppo.server.enip.get_attribute
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/cpppo/server/enip/get_attribute.py", line 376
    rsp,ela		= client.await( connection, timeout=self.timeout )
                          ^
SyntaxError: invalid syntax
```

NOTE: if you have suggestion on how change that function name and you are agree to introduce an API change to accomodate Python 3.7 I can take care of that change.

**Comment by MPW-waters** [comment]
Is this fix (#57) planned to be incorporated in order to support 3.7?

**Comment by mjbrisebois** [comment]
This is now fixed in version 4.0.4
